pyproject.toml

```diff
@@ -29,7 +29,7 @@ dependencies = [
     "pika >= 1.2.0",
     "dataset",
     "pymongo > 3.0",
-    "sdx-pce @ git+https://github.com/atlanticwave-sdx/pce@3.1.0.dev8",
+    "sdx-pce @ git+https://github.com/atlanticwave-sdx/pce@292-l2vpn-request-with-nni-endpoints",
```

Comment:
Leaving a comment here as a place holder for you to remember to create the proper tag and change here